lib/hammer_cli_foreman/report_template.rb

```diff
@@ -77,6 +77,85 @@ def request_params
       build_options
     end
 
+    class ScheduleCommand < HammerCLIForeman::CreateCommand
+      command_name "schedule"
+      action :schedule_report
+
+      option '--inputs', 'INPUTS', N_('Specify inputs'),
+        :format => HammerCLI::Options::Normalizers::KeyValueList.new
+      option '--wait', :flag, _('Turns a command to be active, wait for the result and download it right away')
+      option '--path', "PATH", _("Path to directory where downloaded content will be saved. Only usable if wait is specified"),
+        :attribute_name => :option_path
+
+      def request_params
+        params = super
+        params['input_values'] = option_inputs || {}
+        params
+      end
+
+      def execute
+        data = send_request
+        if option_wait?
+          poll_for_report(data)
+        else
+          puts data['job_id']
```

Comment:
Printing job id only is not very user-friendly approach to tell user what's happening now, isn't it? I mean, what do you think about doing something like `print_message(_('The report has been scheduled. Job ID: %{job_id}') % { job_id: data['job_id'] })`?

Reply:
Yeah, sounds much more useful :+1: thanks.